ui/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
is this default behaviour helpful for us?

Reply:
I see it with fearful eyes.

Reply:
I understand, this is just only for this PR, as I would like to resolve `react-codemirror2` first and the fact that the codebase will require a refactor based on this.

I only include this temporarily so I can still see the dashboard locally. Once we resolve this, then I will be able to remove this, when I move onto MUI